grails-app/controllers/au/org/ala/collectory/IptController.groovy

```diff
@@ -137,9 +138,8 @@ class IptController {
         def provider = providerGroupService._get(params.uid)
 
         def username = collectoryAuthService.username()
-        def admin =  collectoryAuthService.userInRole(grailsApplication.config.ROLE_ADMIN)
```

Comment:
this `admin` variable is used later in this function. Please fix.

Reply:
I will remove the admin var used in the code, since it has been checked